docs/onboarding.md

```diff
@@ -165,7 +175,24 @@ That is all for this guide!
 A reminder that this is just a gentle introduction, and the field is still largely an active area of research; we welcome you to join us in exploring the exciting possibilities of reranking with LLMs!
 
 ## Reproduction Log[*](https://github.com/castorini/pyserini/blob/master/docs/reproducibility.md)
+
+The experiments in this guide could slightly vary in results due to the intrinsic randomness of LLMs, and particularly the `vLLM` library.
+Thus, in addition to a log entry like the previous steps of the onboarding path, we also request that you add an entry to the table below indicating the precise numbers you obtained from running the experiments; we would like to keep track of these to better understand the variance from `vLLM`.
+More specifically, we are interested in the `ndcg_cut_10` score for the RankZephyr and FirstMistral models on the DL20 dataset–the two experiments you have just completed.
+
+| RankZephyr DL20 | FirstMistral DL20 |
```

Comment:
How about do unique values here? Either update the frequency of an existing number or add a new entry with frequency of one. WDYT @ronakice ?
Like: | 0.8201 x1 |
         | 0.8197 x5 |